### PR TITLE
List feature

### PR DESCRIPTION
**Pull request by MadayAlcala**
Hi guys I implemented the hook to delete list, please review it.

### CODE REVIEW

**Review by alszla — DISMISSED** [review]

File: src/test/resources/features/List.feature [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Feature: List
 
-  @logout
+  @logout @deleteList
   Scenario: Create new List
     Given The user goes to "login" page
     When The "user" fills the form with email and password
```
Comment (alszla) [inline src/test/resources/features/List.feature]: Shouldn't this line be an And?

Comment (alszla) [reply src/test/resources/features/List.feature]: Also, regarding the line that is after..., I think a "s" is missing at the end of the word: create

Comment (MadayAlcala) [reply src/test/resources/features/List.feature]: Hi Ale, I already reviewed your comments and solved it.

Comment (alszla) [reply src/test/resources/features/List.feature]: Thanks. Keep up the good work!

**Review by alszla — APPROVED** [review]

**Comment by TheYisusGithub** [comment]
Nothing to observe.

**Review by TheYisusGithub — APPROVED** [review]

**Pull request merged**